common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -161,6 +161,16 @@ trait ABTestSwitches {
 
   Switch(
     ABTests,
+    "ab-acquisitions-epic-always-ask-election",
+    "This will guarantee that the epic is always displayed on election stories",
+    owners = Seq(Owner.withGithub("jranks123")),
+    safeState = On,
```

Comment:
Is this supposed to be `On`?

If so, what are the factors we consider when deciding the `safeState`? Are we moving to an 'On by default' convention? (I haven't written an AB test in a while, so no idea how the ecosystem has shifted! 😅)